app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Post;
+
+class PostController extends Controller
+{
+    public function show($id)
+    {
+        if (!is_numeric($id)) {
+            return view('home.index');
+        }
+
+        $data = []; //to be sent to the view
+
+        $post = Post::findOrFail($id);
+        $data["title"] = $post->getTitle();
+        $data["post"] = $post;
+        return view('forum.show')->with("data", $data);
+    }
+
+    public function destroy($id){
+        $post = Post::findOrFail($id);
+        $post->delete();
+        $data["title"] = "Posts list";
+        $data["posts"] = Post::all();
+
+        return view('forum.list')->with("data", $data, 'success', 'Post deleted successfully!');
+    }
+
+    public function list()
```

Comment:
Change list to index

Reply:
It's because of standards :)

Reply:
But I have two views. Index and list. Do you want that we delete index forum view?

Reply:
Combine them. forum/index is an informational page. Actually, you may do /post/index instead and reffer /forum to that page. Do whichever you preffer, but avoid useless views

Reply:
I done it. I think the we should have a route period/show/{id} instead of period/{id}. It is more descriptive.